app/cypress/integration/user-dashboard.spec.js

```diff
@@ -0,0 +1,29 @@
+describe('User dashboard', () => {
+  before(() => {
+    cy.login(
+      Cypress.env('TEST_REPORTER_USERNAME'),
+      Cypress.env('TEST_REPORTER_PASSWORD')
+    );
+  });
+
+  after(() => cy.logout());
```

Comment:
Looks like this needs to be a `beforeEach`, `afterEach`. The snapshot show the index page (https://percy.io/bcgov-cas/cas-ciip-portal/builds/3907163)